release.sh

```diff
@@ -655,6 +655,23 @@ function publish_artifacts() {
   banner "New release published successfully"
 }
 
+# Sets the github release with the highest semver to 'latest'
+function set_latest_to_highest_semver() {
+  # List latest release
+  local releases # don't combine with the line below, or $? will be 0
+  # Support tags in two formats
+  # - knative-v1.0.0
+  # - v1.0.0
+  releases="$(hub_tool -p release | cut -d '-' -f2)"
+  [[ $? -eq 0 ]] || abort "cannot list releases"
+  local last_version="$(echo "${releases}" | grep '^v[0-9]\+\.[0-9]\+\.[0-9]\+$' | sort -r -V | head -1)"
+  local release_id # don't combine with the line below, or $? will be 0
+  [[ $? -eq 0 ]] || abort "cannot get relase id from github"
+  release_id="$(hub_tool api /repos/${ORG_NAME}/${REPO_NAME}/releases/tags/knative-${last_version} | jq .id)"
```

Comment:
This seems to be the other way around

```suggestion
  release_id="$(hub_tool api /repos/${ORG_NAME}/${REPO_NAME}/releases/tags/knative-${last_version} | jq .id)"
  if [[ $? -ne 0 ]]; then
    abort "cannot get relase id from github"
  fi
```